docs/feature_combo.md

```diff
@@ -87,3 +80,32 @@ If you're using long combos, or even longer combos, you may run into issues with
 In this case, you can add either `#define EXTRA_LONG_COMBOS` or `#define EXTRA_EXTRA_LONG_COMBOS` in your `config.h` file.
 
 You may also be able to enable action keys by defining `COMBO_ALLOW_ACTION_KEYS`.
+
+## Keycodes 
+
+You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. 
+
+
+```
+* `CMB_ON` - Turn combo Feature on
```

Comment:
This should be tabulated instead, probably.

Reply:
It could be, but it's 3 keycodes, and that almost feels overkill.

Reply:
Well... https://docs.qmk.fm/#/feature_key_lock

I like the idea of each feature page having common sections like Usage, Keycodes, Caveats etc. with tables of the related keycodes and config options, which would help the docs feel less like a mishmash of different styles.

Reply:
The same thing applies to the enable/disable/toggle functions <https://docs.qmk.fm/#/feature_backlight?id=backlight-functions>